content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,54 @@ title: Custom Metrics
+### Understanding distribution metrics
 
-### Synchronous vs. asynchronous custom metrics
+Custom metrics submitted from Lambda functions are aggregated as [distributions][2], because they are designed to instrument applications, independent of the underlying hosts. You can query the metrics using aggregations: `avg`, `sum`, `max`, `min`, `count`. You can also enable percentile aggregations (p50, p75, p90, p95, p99), and [manage tags][3] for aggregation on the metric summary page.
 
-The Datadog Lambda Library supports submitting custom metrics in Lambda, both synchronously and asynchronously.
+## Creating custom metrics from logs or traces
 
-**Synchronous**: The default behavior. This method submits your custom metrics to Datadog via HTTP periodically (every 10 seconds) and at the end of your Lambda invocation. If the invocation lasts for less than 10 seconds, your custom metrics are submitted at the end of the invocation.
+With log-based metrics, you can record a count of logs that match a query or summarize a numeric value contained in a log, such as request duration. Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. Learn more about creating log-based metrics [here][4]. 
 
-**Asynchronous (recommended)**: It's possible to submit your custom metrics with zero latency overhead **and** have them appear in Datadog in near-real-time. To accomplish this, the Lambda Library either:
-- Publishes your custom metrics to the [Datadog Lambda Extension][4], which will submit your custom metrics directly to Datadog without impacting your function latency.
-- Emits your custom metrics as specially-formatted log which the [Datadog Forwarder][5] parses and submits to Datadog. Logging in AWS Lambda is 100% asynchronous, so this method ensures there is zero latency overhead to your function.
+You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][5].
+## With the Datadog Lambda Extension
 
-### Enabling asynchronous custom metrics
+{{< img src="serverless/serverless_custom_metrics.png" alt="Collecting Custom Metrics from AWS Lambda" >}}
 
-#### With the Datadog Lambda Extension
+Datadog recommends using the [Datadog Lambda Extension][1] to submit custom metrics from supported Lambda runtimes.
 
-Datadog recommends using the Datadog Lambda Extension if your you are using Node.js or Python Lambda runtimes.
+1. Follow the general [serverless installation instructions][6] to configure your Lambda function and install the Datadog Lambda Library and Extension.
+1. If you are not interested in collecting traces from your Lambda function, set the environment variable `DD_TRACE_ENABLED` to `false`.
```

Comment:
Should this be 2. instead of 1?

Reply:
The best practice is to always use `1` and let markdown automatically render the right number.

Reply:
this is a markdown formatted ordered list and our site supports it (note that it's rendering correctly in the preview), so we can leave it this way